runtime/src/main/java/com/dylibso/chicory/runtime/WasiOptions.java

```diff
@@ -0,0 +1,50 @@
+package com.dylibso.chicory.runtime;
+
+import java.io.InputStream;
+import java.io.PrintStream;
+
+public class WasiOptions {
```

Comment:
an idiomatic Java builder pattern will look like this (from the user's perspective):

```java
WasiOptionsBuilder.withStdin(mystdin).withStdout(mystdout).build();
```

Reply:
Is the pattern internally to just have a static class that looks exactly the same as the regular class? seems pretty wacky. https://www.digitalocean.com/community/tutorials/builder-design-pattern-in-java

Is there a better way?

Reply:
It's the same thing but with mutable fields ... no magic...

Reply:
I'm confused, i still need both a builder class and a normal class with the same instance variables and setters right?

Reply:
> instance variables and setters

you need only the constructor and the getters

Reply:
resolved in #166